src/spikeinterface/curation/tests/test_sortingview_curation.py

```diff
@@ -50,23 +48,22 @@
 #     w = sw.plot_sorting_summary(sorting_analyzer, curation=True, backend="sortingview")
 
 #     # curation_link:
-#     # https://figurl.org/f?v=gs://figurl/spikesortingview-10&d=sha1://bd53f6b707f8121cadc901562a89b67aec81cc81&label=SpikeInterface%20-%20Sorting%20Summary
+#     # https://figurl.org/f?v=npm://@fi-sci/figurl-sortingview@12/dist&d=sha1://058ab901610aa9d29df565595a3cc2a81a1b08e5
 
 
 @pytest.mark.skipif(ON_GITHUB and not KACHERY_CLOUD_SET, reason="Kachery cloud secrets not available")
 def test_gh_curation():
     """
     Test curation using GitHub URI.
     """
-    local_path = si.download_dataset(remote_path="mearec/mearec_test_10s.h5")
-    _, sorting = read_mearec(local_path)
+    sorting = generate_sorting(num_units=10)
     # curated link:
-    # https://figurl.org/f?v=gs://figurl/spikesortingview-10&d=sha1://bd53f6b707f8121cadc901562a89b67aec81cc81&label=SpikeInterface%20-%20Sorting%20Summary&s={%22sortingCuration%22:%22gh://alejoe91/spikeinterface/fix-codecov/spikeinterface/curation/tests/sv-sorting-curation.json%22}
-    gh_uri = "gh://SpikeInterface/spikeinterface/main/src/spikeinterface/curation/tests/sv-sorting-curation.json"
+    # https://figurl.org/f?v=npm://@fi-sci/figurl-sortingview@12/dist&d=sha1://058ab901610aa9d29df565595a3cc2a81a1b08e5
+    gh_uri = "gh://chrishalcrow/spikeinterface/remove-mearec-from-testing/src/spikeinterface/curation/tests/sv-sorting-curation.json"
```

Comment:
This is the line that needs to be changed.

Reply:
Will use the curation file in github, which will fail until sorting curation is merged.